packages/data-extensions/README.md

```diff
@@ -0,0 +1,27 @@
+# `@shopify/data-extensions`
+
+This package contains the public type definitions and utilities needed to create a Shopify data extension.
+
+## What is a data extension?
+
+A data extension is a JavaScript-based module that can hook in to client-side behaviors on Shopify’s first party UI surface areas. It differs from a UI extension in that it does not establish a long-lived channel or render any UI on the surface. It runs once and returns data.
+
+Currently, this package contains the API for one surface.
+
+- [`admin`](./src/surfaces/admin)
+
+All extensions, regardless of where they appear in Shopify, make use the same [underlying technology](../../documentation/how-extensions-work.md) and most of the same capabilities (e.g., direct API access, session tokens).
+
+A data extension using “vanilla” JavaScript would be written as follows:
+
+```ts
+import {dataExtension} from '@shopify/data-extensions/admin';
+
+export default dataExtension(
```

Comment:
I think we'd still want the wrapper around the global so we have the space to do additional things around it if needed. We could also call the global `dataExtension` as well. I think the reason why we had `extend` in the ui-extension package is because the wrapper differs between react (reactExtension) vs. vanilla (extension). In the case of data extension we don't have such differences.

cc @henrytao-me @developit  in case they have opinions